frontend/src/pages/LeaderboardPage.tsx

```
@@ -156,304 +126,372 @@ export function LeaderboardPage() {
         const data = JSON.parse(event.data);
         if (data.type === "leaderboard_update") {
-          queryClient.invalidateQueries({ queryKey: ["leaderboard"] });
+          // Invalidate slightly delayed to allow Redis to settle
+          setTimeout(() => {
+            queryClient.invalidateQueries({ queryKey: ["leaderboard"] });
+          }, 500);
         }
-      } catch (err) {
-        console.error("Failed to parse websocket message:", err);
-      }
+      } catch (err) {}
```

ESLint

[error] 'err' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] Empty block statement.

(no-empty)